app/helpers/react_components/mentoring/representations/admin.rb

```diff
@@ -80,8 +80,8 @@ def links
         def is_introducer_hidden = mentor.introducer_dismissed?(INTRODUCER_SLUG)
 
         SORT_OPTIONS = [
-          { value: :most_submissions, label: 'Sort by highest occurance' },
-          { value: :most_recent, label: 'Sort by recent first' }
+          { value: :feedback_added_at_desc, label: 'Most recent feedbacks' },
+          { value: :feedback_added_at_asc, label: 'Least recent feedbacks' }
```

Comment:
Maybe do:

```suggestion
          { value: :most_recent_feedback, label: 'Most recent feedbacks' },
          { value: :least_recent_feedback, label: 'Least recent feedbacks' }
```

I feel like we sorting option should be more "high-level"-ish, less about the actual columns used